tests/jsTests/indexSpec.js

```diff
@@ -1,43 +1,4 @@
 /**
- * Suite : Tests for showing error on incorrect field entry
- * Description : This is a test suite for showing error in case of an invalid field entry
- */
-describe("Tests for showing error on incorrect field entry", function() {
```

Comment:
They are separated to errorSpec.js to enhance modularity. The corresponding js file is also  made cleaner. @MehaKaushik